src/_premake_init.lua

```diff
@@ -1348,6 +1344,13 @@
 	end)
 
 
+	-- 31 January 2017
+
+	api.deprecateValue("flags", "C++11", 'Use `language "C++11"` instead', function(value) end, function(value) end)
+	api.deprecateValue("flags", "C++14", 'Use `language "C++14"` instead', function(value) end, function(value) end)
+	api.deprecateValue("flags", "C90",   'Use `language "C90"` instead',   function(value) end, function(value) end)
+	api.deprecateValue("flags", "C99",   'Use `language "C99"` instead',   function(value) end, function(value) end)
+	api.deprecateValue("flags", "C11",   'Use `language "C11"` instead',   function(value) end, function(value) end)
```

Comment:
I think the suggestion was ultimately the correct one:

```lua
	api.deprecateValue("flags", "C++11", 'Use `language "C++11"` instead', 
	function(value) 
		language "C++11"
	end, 
	function(value) 
		<what to do here> ???
	end)
```

I've tried a few things,
```lua
removelanguage "C++11"
or
language "C++"
```
but they all break...